HW0/P4/P4.py

```diff
@@ -2,6 +2,23 @@
 import matplotlib.pyplot as plt
 from mpl_toolkits.mplot3d import Axes3D
 
+def predictS(A, s_k, a):
```

Comment:
True, though I tend to avoid using globals like this as it can lead to unexpected behaviour.